.github/workflows/vmangos.yml

```diff
@@ -71,20 +71,16 @@ jobs:
         #directory variables
         export ACE_ROOT=$GITHUB_WORKSPACE/ACE_wrappers
         export TBB_ROOT_DIR=$GITHUB_WORKSPACE/tbb
-        #wget
-        choco install -y wget --version 1.20
         #ACE package download
-        wget http://github.com/DOCGroup/ACE_TAO/releases/download/ACE%2BTAO-$ACE_VERSION2/ACE-$ACE_VERSION.zip
+        curl -LOJ http://github.com/DOCGroup/ACE_TAO/releases/download/ACE%2BTAO-$ACE_VERSION2/ACE-$ACE_VERSION.zip
         unzip ACE-$ACE_VERSION.zip
         rm ACE-$ACE_VERSION.zip
         #configuration of ACE header
         echo "#include \"ace/config-win32.h\"" >> $ACE_ROOT/ace/config.h
         #TBB package download
-        wget https://github.com/oneapi-src/oneTBB/releases/download/v$TBB_VERSION/tbb-$TBB_VERSION-win.zip
+        curl -LOJ https://github.com/oneapi-src/oneTBB/releases/download/v$TBB_VERSION/tbb-$TBB_VERSION-win.zip
         unzip tbb-$TBB_VERSION-win.zip
         rm tbb-$TBB_VERSION-win.zip
-        #openssl
-        choco install -y openssl --version=1.1.1.500
```

Comment:
The installation of OpenSSL via Chocolatey was already removed in the Windows development release workflow by #2210 so it makes sense to remove it here as well anyway (and therefore not require Chocolatey for anything).